app/src/main/java/kotlinrestfull/yassinelhadedy/github/com/kotlinrxrestfull/MainActivity.kt

```diff
@@ -0,0 +1,60 @@
+package kotlinrestfull.yassinelhadedy.github.com.kotlinrxrestfull
+
+import android.R
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.widget.ArrayAdapter
+import android.widget.ListView
+import k2rx.yassinelhadedy.github.com.kotlinretrrx.model.api.RetrofitObject
+import kotlinx.collections.immutable.immutableListOf
+import rx.android.schedulers.AndroidSchedulers
+import rx.schedulers.Schedulers
+
+class MainActivity : AppCompatActivity() {
+
+
+
+    var movisadapter: ArrayAdapter<String>?=null
+
+   // var movisadapter:RecycleAdapter?=null
+
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        //   setContentView(R.layout.activity_main)
+        val listview: ListView
+        //val listview:RecyclerView
+       val movies = immutableListOf<String>()
+//       val movies = mutableListOf<String>()
+
+
+
+
+
+
+        listview= ListView(this)
```

Comment:
This line can be combined with the listview declaration above. If done, you do not have to specify the type of listview. Kotlin will infer it.